Tools/CISupport/ews-build/steps.py

```diff
@@ -3205,7 +3205,7 @@ def start(self):
         architecture = self.getProperty('architecture')
         configuration = self.getProperty('configuration')
 
-        if platform in ['wincairo']:
+        if platform in ['win']:
```

Comment:
I think we could make these `== 'win'` right away.

Reply:
Good catch. Fixed. Thank you.